web_src/js/features/images-lightbox.js

```diff
@@ -0,0 +1,7 @@
+import SimpleLightbox from 'simple-lightbox';
```

Comment:
This should be changed to a dynamic `import` that is conditionally performed only if there is a `.js-lightbox` element on the page. That way, the module will only load on pages where it's actually needed. Also, dynamically load the CSS that way.

See https://github.com/go-gitea/gitea/blob/main/web_src/js/features/colorpicker.js for an example how this is done.

Reply:
Thank you for your comment! I will fix the issue next week. I also found a problem with editing a comment...